gubernator/regex.py

```diff
@@ -41,7 +41,7 @@ def containerID(line):
     return re.search(r'ContainerID:([0-9A-Fa-f]*)', line)
 
 def timestamp(line):
-    timestamp_re = re.search(r'(\d\d-?\d\d[T\s]\d\d:\d\d:\d\d\.\d+)', line)
-    if timestamp_re and timestamp_re.group(0):
-        return re.sub(r'(-|T|\s)', "", timestamp_re.group(0))
-    return ""
+    return re.search(r'(\d\d-?\d\d[T\s]\d\d:\d\d:\d\d\.\d+)', line)
+
+def sub_timestamp(line):
+    return re.sub(r'(-|T|\s)', "", timestamp(line).group(0))
```

Comment:
I might be missing some, but these are examples of two types of timestamps I've seen, which is why I had that line:
I0629 17:33:09.813041
2016-07-22T19:01:11.150204523Z